### DISCUSSION TITLE
Title override option

### DISCUSSION THREAD

**Discussion by dweb-x** [dedoc/scramble#214] [Ideas]
Currently, the API title is set to `config('app.name`);)`; could we have a way of overriding this via the scamble config file?

![scramble-title](https://github.com/dedoc/scramble/assets/6488692/d936adf8-ed43-41c7-a925-28eaecb5e4de)

**Comment by romalytvynenko** [comment]
Hey @dweb-x 

That's a nice idea.

Can be a part of the config, under `scramble.ui.title` key.

Feel free to submit a PR for that!

Meanwhile you can achieve it by publishing `docs` view to your app.
